### PR TITLE
Logging cleanup

### PR DESCRIPTION
**Pull request by mmarchetti**

### CODE REVIEW

File: nb_conda/nbextension/envmanager.py [hunk 1]
```diff
@@ -79,7 +76,7 @@ def get_info(env):
 
     def delete_env(self, env):
         output = self._execute('conda env remove -y -q --json -n', env)
-        output = '\n'.join(output.splitlines()[1:]) # discard 'Fetching package metadata...'
+        output = '\n'.join(output.splitlines()[1:])  # discard 'Fetching package metadata...'
```
Comment (malev) [inline nb_conda/nbextension/envmanager.py]: pep8? I like it more than :pizza:

**Comment by malev** [comment]
looking good.
Question, if I want to develop and I'd like to use logs in a debug mode. How can I do that? Is it worth it to document that somewhere?

File: nb_conda/nbextension/envmanager.py (hunk 1 above)
Comment (mmarchetti) [inline nb_conda/nbextension/envmanager.py]: Yeah, I've been using SublimeLinter-flake8 on save. It's just annoying enough with the warnings to get me to fix them.

**Comment by mmarchetti** [comment]
What do you think of configuring the log level via an environment variable? 

`NB_CONDA_DEBUG=1 jupyter notebook`

**Comment by malev** [comment]
:+1:

**Pull request merged**